modules/data-entry/src/main/frontend/src/questionnaire/Questionnaire.jsx

```diff
@@ -126,6 +132,33 @@ let Questionnaire = (props) => {
     });
   }, []);
 
+  let dropdownList = (
+      <List>
+        <ListItem className={classes.actionsMenuItem}>
+          <ExportButton
+              entityData={data}
+              entryPath={data ? data["@path"] : `/Questionnaires/${id}`}
+              entryName={questionnaireTitle}
+              entryType="forms"
```

Comment:
This can be converted back to the real type for better semantics.
```suggestion
              entryType="Questionnaire"
```